force-app/main/default/lwc/rd2Service/rd2Service.js

```diff
@@ -57,6 +57,10 @@ class Rd2Service {
     async save(rd2State) {
         let result;
         try {
+            // TODO: Move this somewhere more appropriate
+            if (rd2State.recurringType === RECURRING_TYPE_OPEN) {
```

Comment:
Move to `getSaveRequest`, don't include plannedInstallments for Open Recurring Type.